vite.config.ts

```diff
@@ -34,8 +34,10 @@ export default defineConfig({
       input: {
         devtools: resolve(pagesDir, 'devtools', 'index.html'),
         panel: resolve(pagesDir, 'panel', 'index.html'),
+        injectedContent: resolve(pagesDir, 'injectedContent', 'index.ts'),
         contentInjected: resolve(pagesDir, 'content', 'injected', 'index.ts'),
         contentUI: resolve(pagesDir, 'content', 'ui', 'index.ts'),
+
```

Comment:
This separate sth, i think you copied it from #329 
I think this blank space is unnecessary

Reply:
Yes, that change came from manually modifying the conflicts, I'll remove it ^^

Reply:
[81b5d02](https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/494/commits/81b5d02fd5ef65accf2cdafcc7f5b7674e01160b)